docs/phase_locked_tcp_streamer.py

```diff
@@ -229,7 +229,9 @@ def Main():
     output = server_socket.accept()[0].makefile('wb')
     with picamera.PiCamera(clock_mode='raw') as camera:
         camera.resolution = (2592/2,1944/2)
-        camera.framerate = args.fps
+        # You may need to tune the PID values to
+        # get other values of framerate to work.
+        camera.framerate = 30
         camera.exposure_mode = 'fixedfps'
```

Comment:
fixedfps exposure mode is a bit of a misnomer - it's not needed for fixed rate, as the frame rate parameter takes precedence anyway.